User 클래스 및 상품 단위 테스트코드에서 에러가 발생했다.

## 버그 내용
> 버그에 대한 내용을 알려주세요.

Jenkins에서 백엔드 빌드 하는 과정에서 에러가 발생했습니다

## 버그 상황
> 어떤 상황에서 버그가 발생했는지 알려주세요.

Jenkins에서 백엔드 빌드 하는 과정에서 에러가 발생했습니다.
확인해보니, 2가지로 인해 에러가 발생했습니다.
- main 패키지 - user - `private  boolean activated = true;`로 되어있어서 `@SuperBuilder`로부터 해당 필드값이 무시되어집니다.
```java
> Task :compileJava
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/main/java/woorifisa/goodfriends/backend/user/domain/User.java:32: warning: @SuperBuilder will ignore the initializing expression entirely. If you want the initializing expression to serve as default, add @Builder.Default. If it is not supposed to be settable during building, make the field final.
    private  boolean activated = true;
                     ^
```
- test 패키지- UserFixture 클래스에서 필요한 객체가 없어서 테스트 실행에서 에러가 발생했습니다.
```java
> Task :compileTestJava FAILED
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/test/java/woorifisa/goodfriends/backend/common/fixtures/ProfileFixtures.java:28: error: cannot find symbol
                .user(코코())
                      ^
  symbol:   method 코코()
  location: class ProfileFixtures
/var/jenkins_home/workspace/Goodfriends-pipeline/backend/src/test/java/woorifisa/goodfriends/backend/common/fixtures/ProductFixtures.java:40: error: cannot find symbol
                .user(코코())
                      ^
  symbol:   method 코코()
  location: class ProductFixtures
2 errors

FAILURE: Build failed with an exception.
```

## 참고 자료
> 참고할 만한 내용 혹은 주소가 있다면 작성해 주세요.

--- a/backend/src/main/java/woorifisa/goodfriends/backend/user/domain/User.java
+++ b/backend/src/main/java/woorifisa/goodfriends/backend/user/domain/User.java
@@ -31,7 +31,7 @@ public class User extends BaseTimeEntity {
 
     private int ban;
 
-    private  boolean activated;
+    private  boolean activated = true;
 
     protected User() {
     }

--- a/backend/src/test/java/woorifisa/goodfriends/backend/common/fixtures/ProfileFixtures.java
+++ b/backend/src/test/java/woorifisa/goodfriends/backend/common/fixtures/ProfileFixtures.java
@@ -22,9 +22,6 @@ public class ProfileFixtures {
 
     public static final String 팬시_계좌번호 = "112233445566778899";
 
-    public static Profile 팬시() {
-        return new Profile(new User(팬시_이메일, 팬시_닉네임, 팬시_프로필), 팬시_모바일_번호, 팬시_주소, 팬시_계좌종류, 팬시_계좌번호);
-
     // 주소
     public static final String 주소1 = "망원동";
     public static final String 주소2 = "상암동";
@@ -40,6 +37,9 @@ public class ProfileFixtures {
     public static final String 계좌번호2 = "22222222";
     public static final String 계좌번호3 = "33333333";
 
+    public static Profile 팬시() {
+        return new Profile(new User(팬시_이메일, 팬시_닉네임, 팬시_프로필), 팬시_모바일_번호, 팬시_주소, 팬시_계좌종류, 팬시_계좌번호);
+    }
     public static Profile 프로필1() {
         return Profile.builder()
                 .user(코코())

